.github/workflows/eslint.yml

```diff
@@ -0,0 +1,25 @@
+name: ESLint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "18"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run ESLint
+        run: npx eslint --ignore-pattern dist/ .
```

Comment:
Better to use the command from package.json

`npm run eslint`

And add this in package.json. Check other comment.

Reply:
added the npm run lint command in the eslint.yml file